templates/default/parallelcluster_supervisord.conf.erb

```diff
@@ -3,7 +3,13 @@
 <% case node['cfncluster']['cfn_node_type'] -%>
 <%# MasterServer -%>
 <% when 'MasterServer' -%>
-<% unless node['cfncluster']['cfn_scheduler'] == 'awsbatch' -%>
+<% if node['cfncluster']['cfn_scheduler'] == 'slurm' -%>
+[program:clustermgtd]
+command = <%= node['cfncluster']['node_virtualenv_path'] %>/bin/clustermgtd
+redirect_stderr = true
+stdout_logfile = /var/log/clustermgtd
```

Comment:
For the new logic we are writing some of the logs to `/var/log` and some others to `/var/log/parallelcluster/`.
What about unifying the behavior by having everything under the latter? (of course only for the new daemons/scripts)